libs/ui/src/display-line/display-line.component.ts

```diff
@@ -30,4 +36,13 @@ export class WizardsDisplayLineComponent {
     }
     return `${this.firstValue} ${this.secondValue}`;
   }
+
+  startEditing() {
+    this.editing.set(!this.editing());
```

Comment:
Dlaczego przeczenie? Jest szansa ze wykona sie funkcja startEdditing() i this.editing bedzie false?